Added: HomePage, OrderPage, TestFaqText, TestOrder, Test Settings

ilupanudan opened the pull request.

ilupanudan committed:
Added: HomePage, OrderPage, TestFaqText, TestOrder, Test Settings

pomegranate282922 reviewed (commented):
Работа хорошая! Все тесты реализованы. применена параметризация.
Замечания:
1. Нейминг методов
2. Поправить локаторы 
3. Остальное по коду

ilupanudan committed:
Fixed:
-Locators in OrderPage, HomePage;
-Changed names in Parameters (TestFaqText, TestOrder);
-Added camelCase in methods;
-Added static for HOMEURL;
-Renamed class TestSettings to BaseTest;

ilupanudan committed:
fix import ChromeDriver and ChromeOptions in BaseTest

pomegranate282922 reviewed (approved):
Отлично, зачет!
